modules/cert-manager-acme-http01.adoc

```diff
@@ -89,34 +89,32 @@ metadata:
   namespace: my-ingress-namespace                                <2>
   annotations:
     cert-manager.io/cluster-issuer: letsencrypt-staging          <3>
-    acme.cert-manager.io/http01-ingress-class: openshift-default <4>
 spec:
-  ingressClassName: openshift-default                            <5>
+  ingressClassName: openshift-default                            <4>
   tls:
   - hosts:
-    - <hostname>                                                 <6>
-    secretName: sample-tls                                       <7>
+    - <hostname>                                                 <5>
+    secretName: sample-tls                                       <6>
   rules:
-  - host: <hostname>                                             <8>
+  - host: <hostname>                                             <7>
     http:
       paths:
       - path: /
         pathType: Prefix
         backend:
           service:
-            name: sample-workload                                <9>
+            name: sample-workload                                <8>
             port:
               number: 80
 ----
 <1> Specify the name of the Ingress.
 <2> Specify the namespace that you created for the Ingress.
 <3> Specify the cluster issuer that you created.
 <4> Specify the Ingress class.
-<5> Specify the Ingress class.
-<6> Replace `<hostname>` with the Subject Alternative Name to be associated with the certificate. This name is used to add DNS names to the certificate.
-<7> Specify the secret to store the created certificate in.
-<8> Replace `<hostname>` with the hostname. You can use the `<host_name>.<cluster_ingress_domain>` syntax to take advantage of the `*.<cluster_ingress_domain>` wildcard DNS record and serving certificate for the cluster. For example, you might use `apps.<cluster_base_domain>`. Otherwise, you must ensure that a DNS record exists for the chosen hostname.
-<9> Specify the name of the service to expose. This example uses a service named `sample-workload`.
+<5> Replace `<hostname>` with the Subject Alternative Name to be associated with the certificate. This name is used to add DNS names to the certificate.
```

Comment:
I see "Subject Alternative Name (SAN)" in other docs. Consider using the acronym.